cmd/fs-v1.go

```diff
@@ -72,15 +72,83 @@ func initMetaVolumeFS(fsPath, fsUUID string) error {
 
 }
 
+// Attempts to migrate old object metadata files to newer format
+//
+// i.e
+//    -------------------------------------------------------
+//    .minio.sys/buckets/<bucket_name>/<object_path>/fs.json
+//    -------------------------------------------------------
+//    .minio.sys/buckets/<bucket_name>/objects/<object_path>/fs.json
+//    -------------------------------------------------------
+//
+func migrateFSObjectMetadataDir(fsPath, fsUUID string) error {
```

Comment:
Yes we can even block control-c but .. I think it would make sense to print a message . That would make it easier.